doc/Alerting/Transports.md

```diff
@@ -701,6 +701,74 @@ You need a token you can find on your personnal space.
 | API URL | http://api.smsmode.com/http/1.6/sendSMS.do |
 | Options | accessToken=_PUT_HERE_YOUR_TOKEN_<br/> numero=_PUT_HERE_DESTS_NUMBER_COMMA_SEPARATED_<br />message={{ $msg }} |
 
+## SNMP Traps
+
+SNMP traps are usually the preferred way to integrate a system to a northbound service which correlates several alarms.
+This transport will send traps with information from alarms using TRAPv2 or INFORM PDUs (limited to v2c) in conformance
+to [RFC2578](https://www.rfc-editor.org/rfc/rfc2578).
+
+A new MIB file describing the structures has been contributed by Tigo Technology Center, but any other trap definition could be used.
+
+**Example:**
+
+| Config | Example |
+| ------ | ------- |
+| Transport type | Snmptrap |
+| Destination host | noc.mycompany.com |
+| SNMP Trap transport | UDP |
+| Community | out-community |
+| Trap Definition | TTC-NOTIFICATIONS-MIB::daEvent |
+| PDU | TRAPv2 |
+| Binary path | /usr/bin/snmptrap |
```

Comment:
We have tools to find binaries and net-snmp is already required by LibreNMS.

Reply:
Any pointers?

Reply:
Hope this is the right approach: https://github.com/librenms/librenms/commit/5a7ec3ba1d7f584d8491e1092d5337c6fb0cdf04